prom-keycloak-proxy/overlays/nerc-ocp-obs/deployments/prom-keycloak-proxy/deployment.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: prom-keycloak-proxy
+  namespace: prom-keycloak-proxy
+spec:
+  replicas: 1
+  revisionHistoryLimit: 0
+  strategy:
+    type: Recreate
```

Comment:
Why does this deployment need `type: Recreate`?

Reply:
@larsks It doesn't need to be Recreate, I switched it to RollingUpdate instead.